lib/config/defaults.js

```diff
@@ -608,6 +638,8 @@ const applyResolveDefaults = (resolve, { cache, mjs, webTarget }) => {
  */
 const applyResolveLoaderDefaults = (resolve, { cache }) => {
 	D(resolve, "cache", cache);
+	F(resolve, "conditionNames", () => ["node"]);
```

Comment:
> Does webpack-loader or loader make sense here too?
probably, it would be rarely used, but sounds cool. E.g. `toml-parser` package can provide loader using `exports` field. I will add `loader`.